doc/src/stdlib/strings.md

```diff
@@ -38,7 +38,7 @@ Base.searchindex
 Base.rsearchindex
 Base.contains(::AbstractString, ::AbstractString)
 Base.reverse(::Union{String,SubString{String}})
-Base.replace
+Base.replace(s::AbstractString, pat, f)
```

Comment:
This old method is inconsistent with the new ones. Maybe deprecate it in favor of `pat => f`?

Reply:
Done :)